components/bl-stripe-payment-component/src/index.js

```diff
@@ -0,0 +1,42 @@
+import { useState } from 'react';
+
+import { Elements } from './lib/react-stripe.umd.min.js';
+import { loadStripe } from './lib/stripe.min';
+import { PaymentStatus } from './payment-components/payment-status';
+import { PaymentForm } from './payment-form';
+
+const { cn } = BackendlessUI.CSSUtils;
+
+export default function StripePaymentComponent({ component, eventHandlers }) {
+  const { display, classList, publishableKey } = component;
+
+  const [transactionDetails, setTransactionDetails] = useState({});
+  const [isLoading, setIsLoading] = useState(false);
+
+  // SHOULD BE publishableKey from UI Builder SETTINGS
+  const [stripePromise, setStripePromise] = useState(() => {
+    return loadStripe(publishableKey);
+  });
+
+  if (!display) {
+    return null;
+  }
+
+  return (
+    <div className={ cn(classList, 'bl-customComponent-stripe-payment') }>
```

Comment:
In my opinion, it is better when first there will be a main class ('bl-customComponent-stripe-payment'), and then a classList
`cn('bl-customComponent-stripe-payment', classList)`